qemu/tests/cfg/hv_time.cfg

```diff
@@ -9,14 +9,16 @@
     copy_cmd = "copy /y WIN_UTILS:\hv_tools\%%PROCESSOR_ARCHITECTURE%%\%s c:\"
     delete_cmd = "del /f c:\%s"
     hv_time_flags = hv_time hv_stimer hv_stimer_direct
-    extra_params += " -no-hpet"
+    Host_RHEL.m9:
+        extra_params += " -machine hpet=off"
```

Comment:
This should be wrong usage as it will break the `-machine` setting for the whole vm.

Another parameter `machine_type_extra_params`  can be used in this situation.